src/ape_ethereum/ecosystem.py

```diff
@@ -362,7 +362,7 @@ def decode_receipt(self, data: Dict) -> ReceiptAPI:
         elif "input" in data and isinstance(data["input"], str):
             data["input"] = HexBytes(data["input"])
 
-        block_number = data.get("block_number") or data.get("blockNumber")
+        block_number = data.get("block_number", data.get("blockNumber"))
```

Comment:
non-blocking: what's the issue here?

Reply:
just slightly more correct in the odd world if block 0 had logs.
if we were in that case (which tbh not sure is possible in any Eths), it would use the value of `None` here instead.

so more correctness